components/script/dom/webidls/XR.webidl

```diff
@@ -2,9 +2,9 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
```

Comment:
Should this file be called `XRSystem.webidl`?

Reply:
Yes, that will avoid the compile problem.